src/collections/blog/2023/2023-06-21-meshery-operators/index.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: "What is Meshery Operator"
+subtitle: "Describing Meshery operator, what it is and how it simplifies management"
+date: 2023-06-21 10:30:05 -0530
+author: Karan Thakur
+thumbnail: ./gsoc-wide.webp
+darkthumbnail: ./gsoc-wide.webp
```

Comment:
Let's get another image swapped in here...

Reply:
Done